codalab/server/rest_server.py

```diff
@@ -73,10 +82,20 @@ def run_rest_server(manager, debug, num_workers):
     host = manager.config['server']['rest_host']
     port = manager.config['server']['rest_port']
 
-    install(SaveEnvironmentPlugin(manager.model(), manager.bundle_store()))
+    install(SaveEnvironmentPlugin(manager.model(), manager.bundle_store(), manager.config))
     install(CheckJsonPlugin())
     install(LoggingPlugin())
 
+    # Import views
+    import codalab.rest.login
+    import codalab.rest.users
+    from codalab.rest.oauth2 import oauth2_app
+    import codalab.rest.example
+
+    # Mount OAuth2 endpoints
+    oauth2_app.install(SaveEnvironmentPlugin(manager.model(), manager.bundle_store(), manager.config))
```

Comment:
Why does this need to be an app? I have an issue with the fact that you're installing this plugin again.